packages/openapi3/test/tsp-openapi3/parameters.test.ts

```diff
@@ -152,4 +230,36 @@ describe("converts top-level parameters", () => {
     });
     expect(Foo.properties.get("foo")?.type).toBe(serviceNamespace.scalars.get("Foo"));
   });
+
+  ["model", "interface", "namespace", "hyphen-name"].forEach((reservedKeyword) => {
```

Comment:
i don;'t know if it really helps vitest but doing `it.each([...])("escapes invalid names: %s", ` is the recommended way